addons/binding/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/handler/KonnectedHandler.java

```diff
@@ -157,6 +174,19 @@ public void handleWebHookEvent(KonnectedModuleGson event, String sentAuth) {
         });
     }
 
+    private void checkConfigruation() throws ConfigValidationException {
+        KonnectedConfiguration testConfig = getConfigAs(KonnectedConfiguration.class);
+
+        if ((config.hostAddress == null) && (callbackIpAddress == null)) {
```

Comment:
shouldnt this be `testConfig.hostAddress == null`, here?

Reply:
yes it should be, :-)